manager/integration/tests/test_upgrade.py

```diff
@@ -216,6 +230,55 @@ def test_upgrade(longhorn_upgrade_type,
         write_pod_volume_data(core_api,
                               sspod_info['pod_name'],
                               sspod_info['data'])
+
+    # create custom resources
+    # orphan
+    create_orphaned_directories_on_host(
+        client.by_id_volume(pod_volume_name),
+        ["/var/lib/longhorn"],
+        1)
+    # snapshot and backup
+    backup_stores = get_backupstores()
+    if backup_stores[0] == "s3":
+        set_backupstore_s3(client)
+    elif backup_stores[0] == "nfs":
+        set_backupstore_nfs(client)
+        mount_nfs_backupstore(client)
+    backup_vol_name = "backup-vol"
+    backup_vol = create_and_check_volume(client, backup_vol_name, 2, SIZE)
```

Comment:
Can we use already created volume in the test to create snapshot and backup?

Reply:
We can refine this later once a better code structure proposed in https://github.com/longhorn/longhorn/issues/6249#issuecomment-1626356884 has been figured out.